src/watertap_contrib/reflo/property_models/air_water_equilibrium_properties.py

```diff
@@ -0,0 +1,1805 @@
+                return b.collision_molecular_separation_phase_comp[
+                    p, j
+                ] == b.collision_molecular_separation_param * molar_vol ** (
+                    b.collision_molecular_separation_exp
+                )
+
+        self.eq_collision_molecular_separation_phase_comp = Constraint(
+            self.params.vap_set, rule=rule_collision_molecular_separation_phase_comp
+        )
+
+    def _collision_molecular_separation(self):
+        def rule_collision_molecular_separation(b, j):
+            return 0.5 * (
+                b.collision_molecular_separation_phase_comp["Vap", j]
+                + b.collision_molecular_separation_phase_comp["Vap", "Air"]
+            )
+
+        self.collision_molecular_separation = Expression(
+            self.params.solute_set, rule=rule_collision_molecular_separation
+        )
+
+    def _collision_function_comp(self):
+
+        self.collision_function_comp = Var(
+            self.params.solute_set,
+            initialize=0.1,
+            units=pyunits.dimensionless,
+            doc="Collision function",
+        )
+
+        def rule_collision_function_comp(b, j):
+            return b.collision_function_comp[j] == 10 ** (
+                b.collision_function_zeta_comp[j]
+            )
+
+        self.eq_collision_function_comp = Constraint(
+            self.params.solute_set, rule=rule_collision_function_comp
+        )
+
+    def _collision_function_zeta_comp(self):
+
+        self.collision_function_zeta_param_A = a = Param(
+            initialize=-0.14329,
+            units=pyunits.dimensionless,
+            doc="Collision function zeta equation - A parameter",
+        )
+
+        self.collision_function_zeta_param_B = b_ = Param(
+            initialize=-0.48343,
+            units=pyunits.dimensionless,
+            doc="Collision function zeta equation - B parameter",
+        )
+
+        self.collision_function_zeta_param_C = c = Param(
+            initialize=0.1939,
+            units=pyunits.dimensionless,
+            doc="Collision function zeta equation - C parameter",
+        )
+
+        self.collision_function_zeta_param_D = d = Param(
+            initialize=0.13612,
+            units=pyunits.dimensionless,
+            doc="Collision function zeta equation - D parameter",
+        )
+
+        self.collision_function_zeta_param_E = e = Param(
+            initialize=-0.20578,
+            units=pyunits.dimensionless,
+            doc="Collision function zeta equation - E parameter",
+        )
+
+        self.collision_function_zeta_param_F = f = Param(
+            initialize=0.083899,
+            units=pyunits.dimensionless,
+            doc="Collision function zeta equation - F parameter",
+        )
+
+        self.collision_function_zeta_param_G = g = Param(
+            initialize=-0.011491,
+            units=pyunits.dimensionless,
+            doc="Collision function zeta equation - G parameter",
+        )
```

Comment:
Conventionally, I think these params would typically go on the param block. Any particular reason why you did it this way, other than convenience while developing? 

I suppose the only reason it matters is that one would expect to access parameters from the param block, and that'd be pretty consistent with most things in IDAES/WaterTAP, if my recollection is correct.

Reply:
I guess the other advantage of doing it this way is that parameters associated with on-demand props wouldn't be built unnecessarily, and building these on the param block would be less straight-forward if you want to also build on-demand.

Reply:
Both of those reasons are pretty much it. 

And I suppose I'd push back a bit on this convention.  I don't see the advantage/logic behind putting them on the param block and them _always_ being built, but maybe I am missing something. It makes more sense to me to build property-specific params when the on-demand property is built. I would liken this to how we started putting unit-specific costing parameters only in unit-specific costing packages so they weren't always built and cluttering up calls to ``.display()`` or sifting through components via ``.component_objects()`` calls (maybe I am the only one that does that regularly though...)

But I definitely won't die on this hill. I am not opposed to putting them on the param block (or making them Vars, which appears to also be convention).

Reply:
I won't die on this hill either. There are mountains to climb! =)